getting-started/gopher-run/cmd/predict.sh

```diff
@@ -0,0 +1,44 @@
+PROJECT_ID="maralder-start"
+BUCKET_NAME="maralder-start-ml"
```

Comment:
Please use normal env variables rather than inlining the project and bucket name.